livekit-plugins/livekit-plugins-azure/livekit/plugins/azure/tts.py

```diff
@@ -158,6 +162,6 @@ def write(self, audio_buffer: memoryview) -> int:
             samples_per_channel=audio_buffer.nbytes // 2,
         )
 
-        audio = tts.SynthesizedAudio(text="", data=audio_frame)
+        audio = tts.SynthesizedAudio(segment_id=nanoid(), frame=audio_frame)
```

Comment:
does the single frame represent the entire segment? could we have it instantiate a new id if not already set? would be nice to have a default behavior

Reply:
ignore this PR, I was too tired..